hbw/app/assets/stylesheets/hbw/brand_variables.sass

```diff
@@ -65,3 +65,184 @@
   --base-select-dropdown-indicator-color-hover: #333
   --base-select-dropdown-indicator-color-focus: #666
   --base-select-dropdown-indicator-width: 36px
+
+  //Fonts 
+  --base-hbw-default-font-family: "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif  
+
+  //Panel title
+  --base-hbw-panel-title-font-family: var(--base-hbw-default-font-family)
```

Comment:
IMO we should keep the same family through all elements, don't turn widget into patchwork.